### PR TITLE
🌱 addressed previous PR comments

### PR DESCRIPTION
**Pull request by nirrozenbaum**
## Summary

addressed minor comments written by @MikeSpreitzer in #2384.

## Related issue(s)

Fixes #

### CODE REVIEW

File: docs/content/direct/packaging.md [hunk 1]
```diff
@@ -227,7 +227,7 @@ By our development practices and not doing any manual hacking we maintain the as
 
 The [release process](release.md) builds and publishes that container image.
```
Comment (MikeSpreitzer) [inline docs/content/direct/packaging.md:228]: This file should also be updated to talk about the newly introduced image. Does not have to be done in this PR.

**Review by MikeSpreitzer — APPROVED** [review]
/lgtm
/approve
Thanks!

File: docs/content/direct/packaging.md (hunk 1 above)
Comment (nirrozenbaum) [reply docs/content/direct/packaging.md:228]: @MikeSpreitzer right. atm we have no new image. the newly introduced image will be done in a followup PR that adds helm chart. this file will be updated accordingly in the same PR.

**Pull request merged**